eclair-node/src/main/scala/fr/acinq/eclair/Boot.scala

```diff
@@ -39,7 +43,29 @@ object Boot extends App with Logging {
     val setup = new Setup(datadir)
     plugins.foreach(_.onSetup(setup))
     setup.bootstrap onComplete {
-      case Success(kit) => plugins.foreach(_.onKit(kit))
+      case Success(kit) =>
+        plugins.foreach(_.onKit(kit))
```

Comment:
I would initialize the plugins after the API is up (imo initializing external things like plugins should always come last)

Reply:
I agree that plugins initialization should happen last but to be fair our plugin initialization is already interleaved with the bootstrap,  we first call `onSetup` then `onKit`.